code/parachain/frame/pablo/build.rs

```diff
@@ -0,0 +1,197 @@
+#![allow(clippy::disallowed_methods)] // temporarilly
+
+use std::{env, fmt::Write, fs, iter, path::PathBuf};
+
+use syn::{Generics, Item, ItemFn, ItemMod, ReturnType, Signature, Visibility};
+
```

Comment:
some comment what this script does would be nice

Reply:
yeah, working on those right now. once it's ready i'll move the PR out of draft

Reply:
updated the PR description with a link to some documentation on what the build script does and how it works :+1: